src/systems/filecoin_mining/sector/posting.id

```diff
@@ -33,51 +26,6 @@ type PoStType enum {
     SurprisePoSt
 }
 
-// TODO: refactor these types to get rid of the squishy optional fields.
```

Comment:
The below changes will likely lead to some broken links in the markdown. We'll deal with it as it comes up...

Reply:
Yes, I have a follow-up to re-link the new code.